index.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title id="title">Elon Musk</title>
+        <link rel="stylesheet" href="style.css">
+    </head>
+    
+    <body>
+
+        <div id="main">
+            <h1>Elon Musk</h1>
```

Comment:
It would be nice if you had placed the h1 tag inside a header tag. It would make you code semantic and help you keep your code clean.